frame/staking/src/lib.rs

```diff
@@ -2144,12 +2172,17 @@ impl<T: Config> Module<T> {
 				},
 			}
 
-			// new era.
-			Self::new_era(session_index)
+			// New era.
+			let maybe_new_era_validators = Self::try_trigger_new_era(session_index, is_genesis);
+			if maybe_new_era_validators.is_some() && matches!(ForceEra::get(), Forcing::ForceNew) {
+				ForceEra::kill();
```

Comment:
Why do we do this? 

Should it be set to something explicitly or is there a default?

Reply:
ForceNew should only force one single new era, the era after shouldn't be forced, thus we reset ForceEra to the default behavior: NotForcing. I guess writing NotForcing can make it more explicit indeed.

Reply:
> I guess writing NotForcing can make it more explicit indeed.

yeah, would help new people like me at least :P

Reply:
yes done from Kian :-)